go.mod

```diff
@@ -1 +1,5 @@
 module github.com/newrelic/newrelic-telemetry-sdk-go
+
+go 1.14
+
+require github.com/google/uuid v1.1.1
```

Comment:
In general our goal as been to not introduce new dependencies - for example, to create GUIDs in the Go Agent, we have [our own method](https://github.com/newrelic/go-agent/blob/7b8be5e84d2f994b52fad23bf554af11e57fe807/v3/internal/trace_id_generator.go#L52). Could something like that be used here as well?

Reply:
Sure, this could be refactored to do that, not sure I follow why though. Re-implementing a solution seems much more problematic / support burden / error prone than adding a dependency.

Reply:
The reasoning as I understand it is both so the library remains small, and doesn't take long for users to download and doesn't increase the size of their app, and also so that we don't need to worry about dependency versioning  - if, for example, the user already is using this UUID library, but a different version in their app - there could be issues.  

Happy to talk more about this when we chat tomorrow as well!